lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1409,6 +1413,7 @@ instance Buildable ProtocolParameters where
     build pp = blockListF' "" id
         [ "Decentralization level: " <> build (pp ^. #decentralizationLevel)
         , "Transaction parameters: " <> build (pp ^. #txParameters)
+        , "K parameter: " <> build (pp ^. #desiredNumberOfStakePools)
```

Comment:
I'd actually go for "desired number of pools" here too. A bit clearer since there are at least 3 things called `k` in the shelley specs ^^"

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1827/commits/499d4eb76bf2bac9b957940c62e357ffbb4a27d1